backend/src/types/buildingData.ts

```diff
@@ -0,0 +1,6 @@
+type BuildingData = {
+    id: string | null | undefined;
```

Comment:
Nää varmaa johtuu jostai googlekirjaston oikuista, mut vähän hassua et kaikki data voi olla nulleja.  Typescriptin hyödyt vähän heikkenee. 

Tähän ei kyllä oo nyt aikaa tehä muutoksia, mutta olis periaatteessa hyvä että nää mätchäis apidokumentin tyyppejä myös kenttien pakollisuudessa. Tällön frontissa vois myös olettaa tiettyjen kenttien olemassaolon. Se vaatii kyllä sit lisäkoodaamista että validoi ne tyypit sen jälkeen kun ne saa googlelta.

Emt turhaa lätinää mutta halusin mainita.

Reply:
Joo siis mun **muistaakseni** noi tulee suoraan sen Googlen mukaan, käytännössä mun käsityksen mukaan voidaan olla varmoja et noi kaikki kentät kuitenkin on olemassa, jos me on saatu building🤔